indico/web/templates/_outdated_browser.html

```diff
@@ -6,12 +6,27 @@ <h6>Your browser is out of date!</h6>
 </div>
 
 <script>
-    $(document).ready(function() {
+    function addLoadEvent(func) {
+        const oldonload = window.onload;
+        if (typeof window.onload != 'function') {
```

Comment:
`!==`

Reply:
Can't we just use `addEventListener`? Or do some old browsers not support this? (we don't care about IE6)